src/app/shared/components/ds-forms/base-input/base-input.component.ts

```diff
@@ -18,7 +25,21 @@ export class BaseInputComponent {
 
   @Output() inputChange = new EventEmitter<any>();
 
-  constructor() {}
+  public subscriptions = new Subscription();
+
+  public controlInitialized = new BehaviorSubject(false);
+
+  constructor(
+  ) {}
+
+  ngOnInit() {
```

Comment:
`BaseInputComponent` needs to implement `OnInit`

Reply:
Implemented